### PR TITLE
Update RateThisApp.java

### PR DESCRIPTION
**Pull request by fiftysoft**
Add new options to change title of later , No,Thanks , Rate Now buttons

### CODE REVIEW

File: ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java [hunk 1]
```diff
@@ -173,9 +173,12 @@ public static void showRateDialog(final Context context, int themeId) {
     private static void showRateDialog(final Context context, AlertDialog.Builder builder) {
         int titleId = sConfig.mTitleId != 0 ? sConfig.mTitleId : R.string.rta_dialog_title;
         int messageId = sConfig.mMessageId != 0 ? sConfig.mMessageId : R.string.rta_dialog_message;
+        int CancelButtonID = sConfig.CancelButton != 0 ? sConfig.CancelButton : R.string.rta_dialog_cancel;
```
Comment (kobakei) [inline ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java]: Rename to `cancelButtonId`

File: ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java [hunk 2]
```diff
@@ -270,6 +273,9 @@ private static void log(String message) {
         private int mCriteriaLaunchTimes;
         private int mTitleId = 0;
         private int mMessageId = 0;
+        private int RateButton = 0;
+        private int ThanksButton = 0;
+        private int CancelButton = 0;
```
Comment (kobakei) [inline ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java]: Rename to `mCancelButton`

**Comment by kobakei** [comment]
Seems to work fine.
Please update to match the coding style of the library.

**Comment by fiftysoft** [comment]
Done !

File: ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java [hunk 3]
```diff
@@ -270,6 +273,9 @@ private static void log(String message) {
         private int mCriteriaLaunchTimes;
         private int mTitleId = 0;
         private int mMessageId = 0;
+        private int rateButton = 0;
```
Comment (kobakei) [inline ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java]: `mRateButton` is preferred than `rateButton` for coding style.

File: ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java [hunk 4]
```diff
@@ -303,6 +309,30 @@ public void setTitle(int stringId) {
         public void setMessage(int stringId) {
             this.mMessageId = stringId;
         }
+        
+        /**
+         * Set message string ID.
```
Comment (kobakei) [inline ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java]: Fix comment.
ex. Set later button string ID

**Comment by kobakei** [comment]
@fiftysoft I wrote two comments about coding style and comment. Please fix them.

**Comment by fiftysoft** [comment]
Done!

File: ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java [hunk 5]
```diff
@@ -173,9 +173,12 @@ public static void showRateDialog(final Context context, int themeId) {
     private static void showRateDialog(final Context context, AlertDialog.Builder builder) {
         int titleId = sConfig.mTitleId != 0 ? sConfig.mTitleId : R.string.rta_dialog_title;
         int messageId = sConfig.mMessageId != 0 ? sConfig.mMessageId : R.string.rta_dialog_message;
+        int cancelButtonID = sConfig.mCancelButton != 0 ? sConfig.mCancelButton : R.string.rta_dialog_cancel;
+        int thanksButtonID = sConfig.mThanksButton != 0 ? sConfig.mThanksButton : R.string.rta_dialog_no;
+        int rateButtonID = sConfig.mRateButton != 0 ? sConfig.mRateButton : R.string.rta_dialog_message;
```
Comment (kobakei) [inline ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java]: `rta_dialog_message` seems wrong. `rta_dialog_ok` would be correct ID.

Comment (fiftysoft) [inline ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java]: I change it  ,thanks for alert

**Pull request merged**